src/store/embalm/actions.ts

```diff
@@ -246,6 +246,15 @@ export function deselectArchaeologist(address: string): EmbalmActions {
   };
 }
 
+export function setSelectedArchaeologists(archaeologists: Archaeologist[]): EmbalmActions {
```

Comment:
I'm not particularly inclined either way, and you may have already considered this, but:

`setSelectedArchaeologists(archaeologists: Archaeologist[])` might be refactored to `clearSelectedArchaeologists()` since it is only ever used in that capacity.